net/graphsync_fetcher.go

```diff
@@ -79,37 +89,75 @@ const recursionMultiplier = 4
 //
 // See: https://github.com/filecoin-project/go-filecoin/issues/3175
 func (gsf *GraphSyncFetcher) FetchTipSets(ctx context.Context, tsKey types.TipSetKey, from peer.ID, done func(types.TipSet) (bool, error)) ([]types.TipSet, error) {
```

Comment:
We should change the name "from" to something more meaningful at this point in the callstack.  `fetchPeer`, `peerToRequest`, `firstRequest`, I can't think of anything really good but those are some ideas.